modules/unsupported/mbtiles/pom.xml

```diff
@@ -39,26 +39,22 @@
   </licenses>
 
 
-  <!-- =========================================================== -->
-  <!--     Developers and Contributors                             -->
-  <!-- =========================================================== -->
-  <developers>
-    <developer>
-      <id>YOURID</id>
-      <name>YOUR NAME</name>
-      <email>you@server.org</email>
-      <organization>University, Organization or Company</organization>
-      <roles>
-        <role>Module Maintainer</role>
-        <role>Java Developer</role>
-      </roles>
-    </developer>
-  </developers>
```

Comment:
Please fill in this information, I think community modules should have a contact point

Reply:
Its not going to be me.

Reply:
This module has no one looking after it, removing the section was the right call...